kernelcare/manifest.json

```
@@ -0,0 +1,33 @@
+{
+  "display_name": "kernelcare",
+  "maintainer": "schvaliuk@cloudlinux.com",
+  "manifest_version": "1.0.0",
+  "name": "kernelcare",
+  "metric_prefix": "kernelcare.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "A datadog integration to collect kernelcare metrics",
+  "guid": "8b35942d-40cd-4c86-b584-af1837ea67ca",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-kernelcare Integration",
+  "categories": [
+    ""
```

```suggestion
    "security",
    "orchestration",
    "os & system"
```
See some categories that integrations can be filtered by: https://docs.datadoghq.com/integrations/

---

added only "security", "os & system" - it relates to us
"orchestration" not related to us